### ISSUE TITLE
[Feature Request] Add InlineCreate Button Label variable to Backpack 5 Pro

### ISSUE THREAD

**Issue by margarizaldi** [Laravel-Backpack/ideas#139]
Just to give even more flexibility, we should be able to define custom label for inline create button, as "Add" does not always perfectly fit to every add operation.

There should be a kind of `inline_create_button_label` variable, with the default `trans('backpack::crud.add')`.

I think the code can be done in minutes 🤪. So why not?

**Comment by pxpm** [comment]
Hello @margarizaldi 

I think it's reasonable, it's something people already asked for and I've needed it myself sometimes, but hey, I settled on `add is good enough` and moved on 🙃 

The only way to do it non-breakable is by using a config in the field like you suggested, I guess `add_button_label` would be enough since the inline configuration is contained inside the `inline_create => [add_button_label => smt]` attribute.

I think the correct way to fix this would be to use `crud.inline_add` as translation key, but doing it now would be a breaking change so we are stuck with the field inline config if we want to move this foward before the next version. 

Since more people asked for it already, I've submitted the PR to discuss this with @tabacitu . It's not the time it takes to create the PR, is the way we endup implementing things, we need to keep Backpack maintainable for the foreseen years. 

For you to better understand what I am talking about, in this simple scenario, if you are having the add button configurable in the field, you will want to have the other operation buttons (save, cancel) also configurable in field, happens that the save/cancel buttons are loaded in the ajax request for the related crud, so you don't have the field available anymore, to provide the same functionality we would need to pass the button names in the ajax request so they get loaded, and if we start doing things like that we will be burried in technical debt. 

Like I said, PR is created, I will be talking about this with @tabacitu !

Thanks 👍
